bogota-july-2018/JSCalculator/assets/js/script.js

```diff
@@ -0,0 +1,81 @@
+
+var operations = {
+	"+": (num1, num2) => num1 + num2,
+	"-": (num1, num2) => num1 - num2,
+	"*": (num1, num2) => num1 * num2,
+	"/": (num1, num2) => num2 !== 0 ? num1/num2 : "Error!, use Clear button",
+	"":(num1, num2) => num2
+}
+
+var keyCodes = {
+	"106": "*",
+	"107": "+",
+	"109": "-",
+	"111": "/"
+}
+
+var res = 0;
+var first = true;
+var equal = false;
+var opc = "";
+var num2 = 0;
+
+var input = document.querySelector("#input-numbers");
+var output = document.querySelector("output");
+var options  =document.querySelectorAll(".operator");
+var equals = document.querySelector("#equals");
+var clear = document.querySelector("#clear");
+
+for(var i = 0; i < options.length; i++){
+	options[i].addEventListener("click", function(){
```

Comment:
Adding a LOT of evenListener also is not a good idea, here is a doc talking about event delegation that maybe can help u 
https://davidwalsh.name/event-delegate

Reply:
i know we didn't talk about that :)